src/initramfs/cmd/init/pkg/system/services/kubeadm.go

```diff
@@ -109,15 +107,7 @@ func (k *Kubeadm) PostFunc(data *userdata.UserData) error {
 	}
 
 	for _, endpoint := range data.Services.Trustd.Endpoints {
```

Comment:
overall lgtm, curious if there's anywhere else that makes use of trustd.endpoints that may need to be aware of the change from `host:port` to just `host`.

Reply:
👍 From my testing (created a cluster using TF in AWS), it is only here.